services/core-web/src/components/Forms/reports/AddReportForm.js

```diff
@@ -249,7 +251,7 @@ export class AddReportForm extends Component {
             />
             {this.state.existingReport &&
               this.state.mineReportSubmissions.filter((x) => x.mine_report_submission_guid).length >
-                0 && (
+              0 && (
```

Comment:
Do we really need this piece?

Reply:
I think the only intentional change here was formatting - unless necessary for the update we didn't change any logic in the way components are rendered.

(The exception here is the removal of NullScreens on all the tables, but that isn't in this file)